cmd/alertmanager/main.go

```diff
@@ -72,6 +74,27 @@ func init() {
 	prometheus.MustRegister(version.NewCollector("alertmanager"))
 }
 
+func newAlertMetricByState(marker types.Marker, st types.AlertState) prometheus.GaugeFunc {
+	return prometheus.NewGaugeFunc(
+		prometheus.GaugeOpts{
+			Name:        "alertmanager_alerts_count",
```

Comment:
The _count suffix is for summaries/histograms time series

Reply:
Removed.